make/run

```diff
@@ -32,10 +32,19 @@ kubectl get storageclass persistent 2>/dev/null || {
     kubectl create -f -
 }
 
-kubectl create --namespace="${NAMESPACE}" \
-    --filename="kube/bosh" \
-    --filename="kube/secrets" \
-    --filename="kube/bosh-task/post-deployment-setup.yml"
+source bin/settings/settings.env
+source bin/settings/network.env
+namespace=${NAMESPACE}
+source bin/settings/kube/hcp-compatibility.env
```

Comment:
It only has `HCP_SERVICE_DOMAIN_SUFFIX`. I thought that was necessary, but maybe I was confused.